.gitignore

```diff
@@ -8,3 +8,5 @@ system-test/secrets.js
 system-test/*key.json
 *.lock
 *-lock.js*
+build
+.DS_Store
```

Comment:
nit: it would be good to sort this list.

Reply:
Done